### DISCUSSION TITLE
Upgraded to 2.0.0 and all of my index pages kinda broke?

### DISCUSSION THREAD

**Discussion by jmdfm** [excid3/madmin#235] [Q&A] [answered]
I'm only seeing what looks like the primary key, created_at and some actions: 

<img width="1135" alt="image" src="https://github.com/user-attachments/assets/e1e33fe8-a8c0-4e31-b9f5-dc50107f9f4b">

I don't see anything in the docs or release notes about something I should upgrade / change. Not sure if this was intended?

Thanks!

**Comment by excid3** [comment, marked as answer]
By design. You choose what you want to display on the index instead of dumping every single column on there by default.

Add `index: true` to the attributes to display them on the index.

A select few are included by default. https://github.com/excid3/madmin/blob/main/lib/madmin/field.rb#L46

**Reply by jmdfm** [reply]
Ah nice! I totally missed that, very cool.
